.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Add Tags to Release
+
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
Interested to know if this will also kick off the ci-cd.yaml flow again since that is also triggering off of push.

Reply:
it was not kicking of the ci-cd because that was on push to `master` and the release only happens on pushing to something starting with `v`.   this is purely going under the assumption our releases will be have that format.

Reply:
no longer using on push to tag